tests/cache/_fixtures.py

```diff
@@ -252,6 +255,7 @@ def f():
                 time.sleep(0.5)
 
         f()
+        # TODO: Is this test doing anything useful, given that we return here?
```

Comment:
**mike bayer** (zzzeek) wrote:

this is a mistake that got committed at some point.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2283